webextension/js/popup.js

```diff
@@ -1,150 +1,335 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
 /* global browser, window, document, localStorage */
+
 const CONTAINER_HIDE_SRC = "/img/container-hide.svg";
 const CONTAINER_UNHIDE_SRC = "/img/container-unhide.svg";
 
-function showPanel(panelSelector) {
-  for (let panelElement of document.querySelectorAll(".panel")) {
-    panelElement.classList.add("hide");
+// Let's set it to false before releasing!!!
+const DEBUG = false;
```

Comment:
Re: "Let's set it to false before releasing!!!" Do we want this to be `true` during development?
Or some flag we can maybe set in about:addons?